mlbgame/data.py

```diff
@@ -1,10 +1,22 @@
+#!/usr/bin/env python
+
 '''
 This module gets the XML data that other functions use.
 It checks if the data is cached first, and if not, 
 gets the data from mlb.com
 '''
+
+from __future__ import print_function
+
+import sys
 import os
-import urllib2 as url
+
+if sys.version_info[0] == 2:
+    from urllib2 import urlopen
+else:
+    from urllib.request import urlopen
```

Comment:
I agree, try using Python 3 and treat Python 2 as the exception.  If there were other libraries being used, I'd also throw in using `six` to handle the different libraries between the versions.